client/components/theme/index.jsx

```diff
@@ -145,7 +143,7 @@ export class Theme extends Component {
 	};
 
 	render() {
-		const { active, blockEditorSettings, price, theme, translate, upsellUrl } = this.props;
```

Comment:
Now that we're not using `blockEditorSettings` anymore, we can remove the HOC below and the prop type declaration.

Reply:
Done